ql/src/java/org/apache/hadoop/hive/ql/io/sarg/ConvertAstToSearchArg.java

```diff
@@ -552,6 +564,10 @@ private static SearchArgument getSearchArgumentFromExpression(Configuration conf
     }
   }
 
+  public static boolean isCompleteConversion(Configuration conf, ExprNodeGenericFuncDesc expression) {
```

Comment:
Could you please add some unit tests to test this method:
https://github.com/apache/hive/blob/master/ql/src/test/org/apache/hadoop/hive/ql/io/sarg/TestConvertAstToSearchArg.java

Reply:
Added asserts to existing tests in TestConvertAstToSearchArg for each unit test.